splice_machine/src/test/test-data/rows_count.csv

```diff
@@ -0,0 +1,3 @@
+"abc","efg"
+"abc","def"
+"abd",null
```

Comment:
can this small dataset reproduce the error count discrepancies for HFile import with sampling?

Reply:
Yes. With sampling the null row will be count twice.